{{ cookiecutter.package_name }}/docs/index.rst

```diff
@@ -6,6 +6,7 @@ This is the documentation for {{ cookiecutter.package_name }}.
 .. toctree::
   :maxdepth: 2
 
+  documentation/installation
```

Comment:
"documentation" dir under "docs" seems a little redundant.
```suggestion
  installation
```

Reply:
I've seen this in a few packages, to bundle documentation about installation, testing, code quality checks, and miscellany about the package itself, not examples or code documentation.